torchdynamo/utils.py

```diff
@@ -182,9 +182,14 @@ def is_numpy_float_type(value):
 
 def istensor(obj):
     """Check of obj is a tensor"""
-    return istype(
-        obj, (torch.Tensor, torch.nn.Parameter, *config.traceable_tensor_subclasses)
+    tensor_list = (
+        torch.Tensor,
+        torch.nn.Parameter,
```

Comment:
This is a torch tensor subclass - does explicitly listing it if we list tensor do anything if we change the istype to isinstance?

Reply:
I think we want to explicitly register which sublcasses we handle, `isinstance` will mean all subclasses are automatically handled in this way. If people start making a lot of subclasses and we aren't confident we can handle them without tweaks I think an explicit list is better, if not I'm okay with `isinstance`.

Reply:
Yeah, it depends on if we allow arbitrary subclasses of ```FakeTensor```, I don't have too much context on this. cc @eellison

Reply:
It seems like we should support isinstance and then have a reason not to :) do tests and benchmarks pass with just isinstance?

Reply:
```istype``` actually calls ```isinstance```